src/seeds.scheduler.cpp

```diff
@@ -5,26 +5,49 @@
 #include <string>
 
 
-bool scheduler::is_ready_to_execute(name operation){
-    auto itr = operations.find(operation.value);
+uint64_t scheduler::is_ready_to_execute(const name & operation, const name & optype, const uint64_t & timestamp) {
+    if (optype == "op"_n) {
+        auto itr = operations.find(operation.value);
 
-    if (itr == operations.end()) {
-        return false;
-    }
-    if(itr -> pause > 0) {
-        print("transaction " + operation.to_string() + " is paused");
-        return false;
+        if (itr == operations.end()) {
+            return 0;
+        }
```

Comment:
it's just safety precaution, but that's true, it can be removed since it the method is not called anywhere else and by the moment we call the operation exists